content/_code-samples/paths/js/paths.ts

```diff
@@ -0,0 +1,49 @@
+import { Client, Payment, RipplePathFindResponse } from 'xrpl'
+
+const client = new Client('wss://s.altnet.rippletest.net:51233')
+
+async function createTxWithPaths(): Promise<void> {
+  await client.connect()
+
+  const { wallet } = await client.fundWallet()
+  const destination_account = 'rKT4JX4cCof6LcDYRz8o3rGRu7qxzZ2Zwj'
+  const destination_amount = {
+    value: '0.001',
+    currency: 'USD',
+    issuer: 'rVnYNK9yuxBz4uP8zC8LEFokM2nqH3poc',
+  }
+
+  const request = {
+    command: 'ripple_path_find',
+    source_account: wallet.classicAddress,
+    source_currencies: [
+      {
+        currency: 'XRP',
+      },
+    ],
+    destination_account,
+    destination_amount,
+  }
+
+  const resp: RipplePathFindResponse = await client.request(request)
+  console.log(resp)
```

Comment:
Please add context to these logs so someone just reading the command line can understand what they're supposed to be.